### ISSUE TITLE
Using the SDK without a prefix

### ISSUE THREAD

**Issue by jmfeurprier** [mapado/rest-client-sdk#44]
Hi,

we are using the SDK (with the Symfony bundle) with a REST API which has no URL prefix, ie, we have end-points like these:

http://domain.tld/articles
http://domain.tld/articles/434856bf-e37e-11e6-b8b0-0242ac110002

In the YML configs, we put these values:

```
            server_url: 'http://domain.tld'
            mappings:
                prefix: ~
                dir: '%kernel.root_dir%/../src/AppBundle/Entity/'
```

On our entities:

```
/**
 * @Rest\Entity(key="articles")
 */
class Article
{
[...]
}
```

But calls made by the SDK look like this (notice the missing slash /):
http://domain.tldarticles/434856bf-e37e-11e6-b8b0-0242ac110002

We then changed the YML property "prefix" from NULL (~) to '/' but then we get this:
http://domain.tld//articles/434856bf-e37e-11e6-b8b0-0242ac110002

We sort of hacked by putting 'http://domain.tld//' (2 slashes) in the server_url and a NULL prefix, but we get many other problems after (entities are converted to identifiers as "articles/...." instead of "/articles/...." for example).

Is the absence of URL prefix supported? If so, how should we configure the SDK?

Thanks in advance.

**Comment by jdeniau** [comment]
I add some test and tried to fix it in #45 

Can you try this branch and tell me if it's OK for you ? 
Just change the version number in your composer file by `dev-fix-emptyPrefix@dev` and run `composer update mapado/rest-client-sdk`.

If it's OK for you, I will merge and drop a new tag.

Thanks !

**Comment by jmfeurprier** [comment]
It seems to be addressing our issue, thanks a lot ! 

We are facing another issue with relations, but I'll post a new ticket.
